src/actions/publish.js

```diff
@@ -67,6 +68,37 @@ export function uploadFailed(error: mixed): Action {
   return { type: 'UPLOAD_FAILED', error };
 }
 
+/**
+ * This function returns a profile token from a JWT token, if the passed string
+ * looks like a JWT token. Otherwise it just returns the passed string because
+ * this would be the hash directly, as returned by a previous version of the
+ * server.
+ * In the future when the server will be migrated we'll be able to remove this
+ * fallback.
+ */
+function extractProfileTokenFromJwt(hashOrToken: string): string {
```

Comment:
I wonder a bit if this function shouldn't go to `jwt.js` with unit tests, tell me what you think.
Right now it's tested through the store tests, but error cases aren't properly tested.

Reply:
I would probably put it in jwt.js, since it's not an action creator, even if it is covered by store tests. There are a lot utility functions tested this way.

Reply:
okay I'll do that, thanks for the feedback